mergify_engine/rules/__init__.py

```diff
@@ -68,6 +69,30 @@ async def get_branch_protection_conditions(
     ]
 
 
+async def get_depends_on_conditions(
+    ctxt: context.Context,
+) -> typing.List[filter.Filter]:
+    conds: typing.List[filter.Filter] = []
+    for pull_request_number in ctxt.get_depends_on():
+        try:
+            dep_ctxt = await ctxt.repository.get_pull_request_context(
+                pull_request_number
+            )
+        except http.HTTPNotFound:
+            description = f"⛓️ *pull request not found* #{pull_request_number}"
+        else:
+            description = f"⛓️ **{dep_ctxt.pull['title']}** [#{pull_request_number}]({dep_ctxt.pull['html_url']})"
```

Comment:
```suggestion
            description = f"⛓️ **{dep_ctxt.pull['title']}** ([#{pull_request_number}]({dep_ctxt.pull['html_url']}))"
```
That looks more like the usual GitHub format.
We might need some function to standardize how we insert pull requests in markdown everywhere (e.g. like in the queue summary).